packages/playback-core/src/index.ts

```diff
@@ -1,19 +1,24 @@
-import mux, { Options, ErrorEvent } from 'mux-embed';
-
-import Hls, { HlsConfig } from 'hls.js';
-import { AutoplayTypes, setupAutoplay } from './autoplay';
+import mux, { ErrorEvent } from 'mux-embed';
+import Hls from 'hls.js';
+import { setupAutoplay } from './autoplay';
 import { MediaError } from './errors';
 import { setupTracks, addTextTrack, removeTextTrack } from './tracks';
-import { isKeyOf, inSeekableRange, addEventListenerWithTeardown } from './util';
-import type { Autoplay, UpdateAutoplay } from './autoplay';
-
-export type ValueOf<T> = T[keyof T];
-export type Metadata = Partial<Options['data']>;
-export type PlaybackEngine = Hls;
-export { mux, Hls, MediaError, Autoplay, UpdateAutoplay, setupAutoplay, addTextTrack, removeTextTrack };
+import { inSeekableRange, addEventListenerWithTeardown, toPlaybackIdParts, getType } from './util';
+import {
```

Comment:
nitpick(blocking): for types, we should try to use `import type` wherever possible if that's all we're importing.